core/renderers/block_rendering_rewrite/measurables.js

```diff
@@ -1,7 +1,29 @@
+/**
```

Comment:
Can you leave this file out?  I'm in the middle of a major rewrite and want to avoid merge conflicts.

Reply:
Done.